activity/activity.info

```diff
@@ -3,9 +3,9 @@ name = Reflection
 activity_version = 14
 license = GPLv3
 bundle_id = org.sugarlabs.RelfectionActivity
-exec = sugar-activity ReflectionActivity.ReflectionActivity
+exec = sugar-activity3 ReflectionActivity.ReflectionActivity
 icon = activity-reflection
 show_launcher = yes
 summary = draw patterns of reflective symmetry
 category = maths game
-repository = https://github.com/walterbender/reflection
\ No newline at end of file
+repository = https://github.com/walterbender/reflection
```

Comment:
maybe the repository could be updated to `sugarlabs/reflection`, maybe not in this PR, but in future :smile:

Reply:
0ffe1e0